<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">

    <ros2_control name="GazeboSimSystem" type="system">
        <hardware>
            <plugin>gz_ros2_control/GazeboSimSystem</plugin>
        </hardware>
        <joint name="palm_x">
            <command_interface name="position">
                <param name="min">-1000</param>
                <param name="max">1000</param>
            </command_interface>
            <state_interface name="position">
                <param name="initial_value">0.0</param>
            </state_interface>
        </joint>
    </ros2_control>

    <gazebo>
        <!-- <plugin
            filename="libignition-gazebo-joint-state-publisher-system.so"
            name="ignition::gazebo::systems::JointStatePublisher">
            <topic>/left_hand/joint_states</topic>
        </plugin> -->

        <!-- Joint state publisher -->
        <!-- <plugin filename="libgz_ros2_control-system.so" name="gz_ros2_control::GazeboSimROS2ControlPlugin">
            <robot_param>left_hand/robot_description</robot_param>
            <robot_param_node>robot_state_publisher</robot_param_node>
            <parameters>$(find teleop_tasks)/config/left_hand_controller.yaml</parameters>
        </plugin> -->
    </gazebo>

</robot>